lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -210,14 +210,22 @@ defmodule ExUnit.CLIFormatter do
     do: colorize([:bright, :black], msg, config)
 
   defp formatter(:diff_delete, msg, config),
-    do: colorize(:red, msg, config)
+    do: colorize(:red, show_whitespace(msg), config)
 
   defp formatter(:diff_insert, msg, config),
-    do: colorize(:green, msg, config)
+    do: colorize(:green, show_whitespace(msg), config)
 
   defp formatter(_,  msg, _config),
     do: msg
 
+  defp show_whitespace(msg) do
+    if String.match?(msg, ~r/\A[ ]+\z/) do
```

Comment:
If we compact multiple spaces back into one fragment we'll need to start using a regex in the formatter as we can no longer pattern match on `" "` fragments due to not knowing the length.